general/environments/environments.md

```diff
@@ -0,0 +1,18 @@
+### System properties
+
+|  Operating System |   |   |   |
+|---|---|---|---|
+| Ubuntu  |  ubuntu-latest  |  [ubuntu](http://www.ubuntu.com) | Free software licenses (mainly GPL)  |
+|  **Database** |   |   |   |
+|  MySQL | 5.5.40  | [http://www.mysql.com/](http://www.mysql.com/)  | GPL (version 2) or proprietary  |
+| **Python**  |   |   |   |
+|  Python | 3.8  | [http://www.python.org/](http://www.python.org/)  | Python Software Foundation License  |
+| Flask  | 1.1.2 | [http://flask.pocoo.org/](http://flask.pocoo.org/) | BSD   |
+| Jinja2  | 2.6  | [http://jinja.pocoo.org/](http://jinja.pocoo.org/ ) |  BSD |
+| NumPy  |  1.7.0 |  [http://www.numpy.org/](http://www.numpy.org/) | BSD-new  |
+| SciPy  | 0.11.0  | [http://www.scipy.org/]([http://www.scipy.org/])  | BSD-new  |
+| **JavaScript / CSS**  |   |   |   |
+| jQuery  | 1.10.2  | [http://jquery.com/]([http://jquery.com/)  | MIT  |
+|Bootstrap   |  3.3.0 | [http://getbootstrap.com/](http://getbootstrap.com/)   | MIT License (Apache License 2.0 prior to 3.1.0)  |
+|  DataTables |  1.10.19 |  [http://www.datatables.net/](http://www.datatables.net/) |  MIT |
+| CKEditor  | 4.13.0  | [http://ckeditor.com/](http://ckeditor.com/)  |  GPL, LGPL and MPL |
```

Comment:
**nitpick:** I'll replace this with a script that fetches this info from guix :)